lv_drv_conf.h

```diff
@@ -123,13 +123,31 @@
 #  define USE_WIN32DRV       0
 #endif
 
+#if USE_WIN32DRV
+/* Scale window by this factor (useful when simulating small screens) */
+#  define WIN32DRV_MONITOR_ZOOM        1
+#endif
+
 /*----------------------------------------
  *  GTK drivers (monitor, mouse, keyboard
  *---------------------------------------*/
 #ifndef USE_GTK
 #  define USE_GTK       0
 #endif
 
+/*----------------------------------------
+ *  Wayland drivers (monitor, mouse, keyboard, touchscreen)
+ *---------------------------------------*/
+#ifndef USE_WAYLAND
+#  define USE_WAYLAND       0
+#endif
+
+#if USE_WAYLAND
+#  define WAYLAND_HOR_RES      480
```

Comment:
I'm not up to speed with LVG 8 yet and it looks like they've got rid of LV_HOR_RES. However shouldn't WAYLAND_HOR_RES match MONITOR_HOR_RES?
I think I'd be tempted to add a define at the top of lv_drv_conf.h, what do you think?

Reply:
I guess that's something to be discussed with whomever maintains those drivers. Since this project is about VSCode, I was reluctant to change anything related to lvgl or drivers; the way I see it, this project should just use it. Maybe contribute to drivers project instead?